chunky/src/java/se/llbit/chunky/renderer/RenderContext.java

```diff
@@ -66,47 +77,55 @@ public int numRenderThreads() {
   }
 
   /**
+   * @param descriptionFileName The name of the scene description file. NOTE: Do not include extension
    * @return Scene description file
    */
-  public File getSceneDescriptionFile(String sceneName) {
-    return getSceneFile(sceneName + Scene.EXTENSION);
+  public File getSceneDescriptionFile(String descriptionFileName) {
+    return getSceneFile(descriptionFileName + Scene.EXTENSION);
   }
 
   /**
+   * @param descriptionFileName The name of the scene description file. NOTE: Do not include extension
    * @return Input stream for the scene description
-   * @throws FileNotFoundException
+   * @throws FileNotFoundException If the file does not exist.
    */
-  public InputStream getSceneDescriptionInputStream(String sceneName) throws FileNotFoundException {
-    return getSceneFileInputStream(sceneName + Scene.EXTENSION);
+  public InputStream getSceneDescriptionInputStream(String descriptionFileName) throws FileNotFoundException {
+    return getSceneFileInputStream(descriptionFileName + Scene.EXTENSION);
   }
 
   /**
+   * @param descriptionFileName The name of the scene description file. NOTE: Do not include extension
    * @return Output stream for the scene description
-   * @throws FileNotFoundException
+   * @throws FileNotFoundException If the file does not exist.
    */
-  public OutputStream getSceneDescriptionOutputStream(String sceneName)
+  public OutputStream getSceneDescriptionOutputStream(String descriptionFileName)
       throws FileNotFoundException {
-    return getSceneFileOutputStream(sceneName + Scene.EXTENSION);
+    return getSceneFileOutputStream(descriptionFileName + Scene.EXTENSION);
   }
 
   /**
-   * @return Input stream for the given scene file
+   * Gets the directory of the given scene file.
+   * @param fileName the filename WITH the extension
+   * @return A File object. NOTE: The file object may not exist yet.
    */
   public File getSceneFile(String fileName) {
-    return new File(config.sceneDir, fileName);
+    ensureSceneDirectory();
+    return new File(sceneDirectory, fileName);
   }
 
   /**
+   * @param fileName the filename WITH the extension
    * @return Input stream for the given scene file
-   * @throws FileNotFoundException
+   * @throws FileNotFoundException If the file does not exist.
```

Comment:
The exception is literally called "file not found" exception. Do we really need to document when we throw it?

Reply:
I know haha, I was just annoyed how IJ was yelling at me to put a description. Plus, I'm just used to putting descriptions on things in javadocs even if the name is obvious. 

I've never seen java leave one of those blank, which was another reason I did it too.